frontend/src/components/AgreementCard.tsx

```diff
@@ -23,7 +23,7 @@ interface IAgreementCardProps {
   title: string;
   updatedAt: Date;
   participants: number;
-  isArchived?: boolean;
+  is_archived?: boolean;
```

Comment:
Let's stick to camelCase instead of kebab_case (that's the convention in the frontend)
```suggestion
  isArchived?: boolean;
```

Reply:
fixed